datasette/app.py

```diff
@@ -899,8 +899,11 @@ async def expand_foreign_keys(self, database, table, column, values):
             fk = [
                 foreign_key
                 for foreign_key in foreign_keys
-                if foreign_key["column"] == column
+                if foreign_key["columns"][0] == column
```

Comment:
because we don't have a strong idea of how do display compound foreign keys in a table view, we have to do this operation in a few places.